### DISCUSSION TITLE
alpha without google

### DISCUSSION THREAD

**Discussion by ghost** [AnySoftKeyboard/AnySoftKeyboard#2693] [Q&A] [answered]
Hi,

I'd like to test the alpha version. But the main reason I discoverd this keyboard is my willing to avoid gafam.
Is there a way to get the alpha without suscribe to Google groups ?

Olivier

**Comment by menny** [comment, marked as answer]
Yes, you can download it from the build-system: https://github.com/AnySoftKeyboard/AnySoftKeyboard/actions/runs/526564925

A new build is done with `imeMaster_alpha_100 deployment`. Just search for the latest one under the Actions/Deployment tab: https://github.com/AnySoftKeyboard/AnySoftKeyboard/actions?query=workflow%3Adeployment

**Comment by ghost** [comment]
Thank you.
I managed to download it. But it can't install  (after process,  I have a message "app not installed). Is there something to setup before? (I precise I can install unknowed apk)
Olivier

9 Feb 2021 18:18:40 Menny Even Danan <notifications@github.com>:

**Reply by menny** [reply]
If you have an already installed ASK, it could be that the signatures are different. The link I sent you uses a publishing signature, it is not compatible with FDroid and most likely not compatible with Play Store installations.

I suggest you backup your settings in ASK, and then uninstall and install the ALPHA. All alphas are signed with the same key so future installs should be upgradable.

**Comment by ghost** [comment]
Tes thank You. I've just uninstalled the beta  and installed the alpha.  Everything works.
Regards

10 Feb 2021 18:35:16 Menny Even Danan <notifications@github.com>:
